AEPMessaging/Tests/TestHelpers/MockMessagingRulesEngine.swift

```diff
@@ -44,9 +44,13 @@ class MockMessagingRulesEngine: MessagingRulesEngine {
 
     var loadPropositionsCalled = false
     var paramLoadPropositionsPropositions: [PropositionPayload]?
-    override func loadPropositions(_ propositions: [PropositionPayload]) {
+    var paramLoadPropositionsClearExisting: Bool?
+    var paramLoadPropositionsPersistChanges: Bool?
```

Comment:
These params `paramLoadPropositionsClearExisting` and `paramLoadPropositionsPersistChanges ` do not need to be optional.

Reply:
i prefer to leave them optional so they will be nil if this mock method is not called